app/src/components/dialog/EditDialog.tsx

```diff
@@ -124,7 +124,7 @@ export const EditDialog = <T extends FormikValues>(props: PropsWithChildren<IEdi
       initialValues={props.component.initialValues}
       enableReinitialize={true}
       validationSchema={props.component.validationSchema}
-      validateOnBlur={props.component.validateOnBlur ?? true}
+      validateOnBlur={props.component.validateOnBlur ?? false}
```

Comment:
Are we sure we want the default for this to be false?

Reply:
Yeah, I've heard complaints that its distracting - not a big deal either way